src/Data/Foldable.purs

```diff
@@ -177,6 +177,8 @@ fold = foldMap id
 -- |
 -- | Note: this function is not generally stack-safe, e.g., for monads which 
 -- | build up thunks a la `Eff`.
+-- | For a stack-safe alternative, see
```

Comment:
Might want to link to the actual function docs, and also note that it only works for `List`?

Reply:
Good point. Will update.